zendframework/zend-validator/ZF2015-03.yaml

```diff
@@ -0,0 +1,17 @@
+title:     Invalid CSRF validation of null or incorrectly formatted token identifiers
+link:      http://framework.zend.com/security/advisory/ZF2015-03
+cve:       CVE-2015-1786
+branches:
+    2.0.x:
+        time:     2015-03-12 13:58:47
+        versions: [>=2.0.0,<2.0.99]
```

Comment:
> I would assume that you can remove the entries for `2.0.x`, `2.1.x` and
> `2.2.x`.

Right, the documentation states that `branches` is a hash of affected
branches, so I also fixed the previous ZendFramework entries.

Thanks for the review!

Regards

David